src/main/java/org/folio/inventory/dataimport/handlers/actions/ReplaceInstanceEventHandler.java

```diff
@@ -143,7 +141,17 @@ public CompletableFuture<DataImportEventPayload> handle(DataImportEventPayload d
         String targetInstanceTenantId = dataImportEventPayload.getContext().getOrDefault(CENTRAL_TENANT_ID, dataImportEventPayload.getTenant());
         Context instanceUpdateContext = EventHandlingUtil.constructContext(targetInstanceTenantId, dataImportEventPayload.getToken(), dataImportEventPayload.getOkapiUrl());
         InstanceCollection instanceCollection = storage.getInstanceCollection(instanceUpdateContext);
-        processInstanceUpdate(dataImportEventPayload, instanceCollection, context, instanceToUpdate, future, targetInstanceTenantId);
+
+        //Check existing Instance source
+        InstanceUtil.findInstanceById(instanceToUpdate.getId(), instanceCollection)
+          .onSuccess(existingInstance -> {
+            LOGGER.info("handle:: Processed Instance jobExecutionId: {}.", dataImportEventPayload.getJobExecutionId());
```

Comment:
At this point Instance is found, but not processed yet - please correct the log message

Reply:
done